packages/rune-games-cli/src/flows/Create.tsx

```diff
@@ -133,16 +159,24 @@ export function Create({ args }: { args: string[] }) {
     if (step === Steps.Creating) {
       onCreate()
     }
-  }, [step, onCreate])
+
+    if (step === Steps.Installing) {
+      onInstall()
+    }
+
+    if (step === Steps.PostInstall) {
+      onPostInstall()
+    }
+  }, [step, onCreate, onInstall, onPostInstall])
 
   return (
     <Box flexDirection="column">
       <Step
         status={step > Steps.Target ? "success" : "userInput"}
         label={
           step > Steps.Target
-            ? `Will create a project in "${targetDir}"`
-            : "Project name"
+            ? `Will create a game in "${targetDir}"`
+            : "Game folder name"
```

Comment:
Would it be better to stick to "directory" here - folder is quite non-dev language.

Reply:
We've been going with @bfelbo on this one for a bit. I'm okay with both :)

Reply:
Don't have a preference either. Sounds good to do "directory" 👍

Reply:
renamed to directory.